streaming-availability-app/pages/api/details.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  const options = {
+    method: "GET",
+    url: "https://ott-details.p.rapidapi.com/gettitleDetails",
+    params: { imdbid: req.query.id },
+    headers: {
+      "x-rapidapi-host": "ott-details.p.rapidapi.com",
+      "x-rapidapi-key": NEXT_PUBLIC_RAPIDAPI_KEY,
+    },
+  };
+
+  axios
```

Comment:
You can use `try...catch` with `await` since you are defining an `async` function.

Reply:
@ahmadbilaldev Can you please fix this issue?